Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM rust:bookworm AS builder
+
+RUN apt-get update
+RUN apt-get install -y curl git build-essential libssl-dev pkg-config 
```

Comment:
For what purpose do we need git here? Since we are not cloning anything perhaps `git` could be removed. Same goes for `curl` I believe.

Reply:
true, removed these two.